common-docs/blocks-embed.md

```diff
@@ -349,6 +349,6 @@ f.contentWindow.postMessage({
 
 * [HTML only](https://jsfiddle.net/L8msdjpu/3/)
```

Comment:
What does "HTML only" mean? - "An HTML render example of an embedded project"?

Reply:
Meaning a basic example without using any kind of JS framework